AuthService/cmd/main.go

```diff
@@ -1,9 +1,21 @@
 package main
 
 import (
-	"fmt"
+	"authservice/jwtproc"
+	"authservice/models"
+	"authservice/parsistence"
 )
 
 func main() {
```

Comment:
nit: This function looks more like a test, than a `main()`. But it's ok if you plan to add https serving capabilities in near future.